elasticapm/instrumentation/packages/graphql.py

```diff
@@ -52,7 +52,7 @@ def call(self, module, method, wrapped, instance, args, kwargs):
         query = args[2]
 
         if "ResolveInfo" == type(query).__name__:
-            if str(query.return_type) in [
+            if str(query.return_type).rstrip('!') in [
```

Comment:
```suggestion
            if str(query.return_type).rstrip("!") in [
```

This is why lint is failing